### ISSUE TITLE
The instruction images are difficult to read

### ISSUE THREAD

**Issue by Vidyut** [jpbouza/BlenRig#4]
Not sure whether it is my eyesight or the screen resolution, but it is hard to follow through the guide, because of its semi-transparent background and small size of text. Increasing display size in blender or zooming in doesn't appear to alter it so I have really tiny text even if I can make everything else big. It would help to be able to see a bigger version of the image or at least make the background opaque so that the scene in the background doesn't add difficulty.

**Comment by jpbouza** [comment]
Hi @Vidyut !
Note that in the rigging assistan panel, there is an options tab. If you modifiy the DPI and image size values there, you will be able to make it look better for your monitor. The only problem is that you need to set those options before you open any of the guides, otherwise you need to restart blender, set the options and then open the guides again.

![image](https://user-images.githubusercontent.com/25966711/235284363-8fd55db4-4470-4f66-837f-d48ffd33cde1.png)

**Comment by Vidyut** [comment]
Facepalm. One place I didn't check. Thanks a ton!
